README.md

```diff
@@ -244,6 +246,14 @@ __AWESOME__ apps counter: __150__ 🎉
 - [x] [GitHub](https://github.com/topjohnwu/Magisk)
 - [x] [Official page](https://topjohnwu.github.io/Magisk/)
 
+### SmartPack-Kernel Manager
```

Comment:
I don't know about this one. There is a rule `Easy to use` and are there any proprietary alternative to this app?